lib/factorial.js

```diff
@@ -0,0 +1,10 @@
+function factorial(num) {
+  let facto = 1;
```

Comment:
looks good - maybe we make this variable name more specific? like `startingFactorial`?